DISCLAIMER

```diff
@@ -0,0 +1,20 @@
+This material was prepared as an account of work sponsored by an agency of the
```

Comment:
I think we can drop the disclaimer file, I intend to do that in the HELICS repo at HELICS 3.0, so is isn't necessary to have NOTICE and DISCLAIMER